backend/resolvers/mutations/show.js

```diff
@@ -47,6 +47,8 @@ export function updateShow (_, args, req) {
     .then((show) => {
       return show.update(args.input, {
```

Comment:
I think the restriction for when a show should be edited should probably get mirrored on the BE as well, since there isn't really much to stop the user from direct urling to the edit page and trying to edit.

Reply:
I do dynamically render the form on the edit page front end depending on whether the submission date is after the entry start. If they direct link into the edit page after entry start it only gives them an <h2> tag that says the window for editing a show has ended.